docker-compose-with-wrappers.yml

```diff
@@ -88,16 +88,50 @@ services:
       - filestorage
       - wps
     environment:
-      appID: 'modelprop-asyncwrapper'
+      appID: 'modelprop-eq-asyncwrapper'
       wrapperClass: 'org.n.riesgos.asyncwrapper.dummy.ModelpropEqWrapper'
       spring.datasource.url: 'jdbc:postgresql://db:5432/postgres'
       spring.datasource.username: 'postgres'
       spring.datasource.password: 'postgres'
       pulsar.pulsarURL: 'pulsar://queue:6650'
       pulsar.inputTopics: ''
       pulsar.orderTopic: 'new-order'
-      pulsar.outputTopic: 'modelprop-success'
-      pulsar.failureTopic: 'modelprop-failure'
+      pulsar.outputTopic: 'modelprop-eq-success'
+      pulsar.failureTopic: 'modelprop-eq-failure'
+      wps.wpsUrl: 'http://riesgos-wps:8080/wps/WebProcessingService'
+      wps.wpsVersion: '2.0.0'
+      wps.process: 'org.n52.gfz.riesgos.algorithm.impl.ModelpropProcess'
+      filestorage.endpoint: http://filestorage:9000
+      filestorage.user: admin
+      filestorage.password: secretpassword
+      filestorage.bucketName: riesgosfiles
+      filestorage.access: http://filestorage:9000/riesgosfiles/
+
+  modelprop_ts_wrapper:
+    image: spring-boot-image
+    build:
+      context: asyncwrapper/dockerfile-spring-boot-baseimage
+    volumes:
+      - './asyncwrapper/target/asyncwrapper-0.0.1-SNAPSHOT.jar:/app.jar'
+    depends_on:
+      - db
+      - queue
+    networks:
+      - db
+      - queue
+      - filestorage
+      - wps
+    environment:
+      appID: 'modelprop-ts-asyncwrapper'
+      wrapperClass: 'org.n.riesgos.asyncwrapper.dummy.ModelpropTsWrapper'
+      spring.datasource.url: 'jdbc:postgresql://db:5432/postgres'
+      spring.datasource.username: 'postgres'
+      spring.datasource.password: 'postgres'
+      pulsar.pulsarURL: 'pulsar://queue:6650'
+      pulsar.inputTopics: ''
```

Comment:
Just for the completeness:

I wondered about listening to the tsunami service before this here starts.
But there is really no constraint that would require to wait.